frontend/components/Countdown.tsx

```diff
@@ -1,18 +1,26 @@
 import styles from '../styles/Home.module.css';
-import { compareAsc } from 'date-fns';
+import { compareAsc, getWeekYear } from 'date-fns';
 import CountDown from './Count.js';
 
-export default function Countdown() {
-  const current_date = new Date();
-  const targetDate = new Date('2023-11-22T00:00:00Z');
-  const variable = compareAsc(current_date, new Date(2023, 10, 5, 2, 0, 0));
+export default function Countdown({ date }: { date: Date }) {
+  const targetDate = new Date('2024-04-22T00:00:00Z');
```

Comment:
change the string inside of new Date to just `2024-04-22` so it's more readable.